falcon/routing/util.py

```diff
@@ -183,3 +195,8 @@ def set_default_responders(method_map):
     for method in COMBINED_METHODS:
         if method not in allowed_methods:
             method_map[method] = na_responder
+
+
+class AltMethodNotFoundError(Exception):
```

Comment:
Sorry to be pedantic, but let's rename this to coincide with the change from "alt" to "suffix".